docs/parser_plugins.md

```diff
@@ -1 +1,210 @@
+Parsers _automate the conversion_ of a file format common to a specific piece of software or a community into the NOMAD format.
+The schema for computational techniques was introduced in the previous section.
+
+The key advantages of the NOMAD schema are summed up in **FAIR**mat's core values:
+
+- **F**indable: a wide selection of the extracted data is indexed in a database, powering a the search with highly customizable queries and modular search parameters.
+- **A**ccessible: the same database specifies clear API and GUI protocols on how retrieve the _full_ data extracted.
+- **I**nteroperable: we have a diverse team of experts who interface with various materials science communities, looking into harmonizing data representations and insights among them. Following the NOMAD standard also opens up the (meta)data to the NOMAD apps eco-system.
+- **R**eproducible: data is not standalone, but has a history, a vision, a workflow behind it. Our schema aims to capture the full context necessary for understanding and even regenerating via metadata.
+
+## Modular Design and Plugins
+
+Historically, our parsers have come always come together as a neat bundle.
+As interest in research data management has become more widespread and recognized across scientific disciplines, NOMAD has adapted along with those interests.
+Nowadays, you can find besides the [central NOMAD cloud service](), also standalone NOMAD Oasis deployments geared towards the needs of individual institutes or research groups.
+
+As such, NOMAD is shifting to a _plugin model_, where each deployment administrator can customize their setup.
+This permits a light-weight base installation and effective data processing pipelines.
+Conversely, it also lowers the threshold for whomever wants to contribute to this open-source community: you can focus on just a single schema or parser.
+The possibilities are as wide as you can make them, we just facilitate the common basis.
+
+In the rest of this page, we are going to take you on a birds-eye overview of how a parser developer approaches these plugins.
+
+### Getting started
+
+Getting hands-on with a plugin is just as easy visiting our [GitHub template]() and _using one in your own codespace_.
+The template will appear almost empty at the start, but following the instructions `README.md`  and during the `cruft` setup will allow you to tune the project to your needs.
+Just a couple of notes on the latter:
+
+- NOMAD uses the Apache 2.0 license. Unless your organization prohibits this, please select the same license for maximal legal compatibility.
+- Parsers require both a `parsers` and a `schema_packages` folder. The reason for the second folder will soon become apparent.
+- We use generically named files like `parsers/myparser.py` as blueprints by copying them to new, more clearly named files. Each file format, its own Python file.
+
+### Managing Entry Points
```

Comment:
> I think this should be moved either down towards the end of this section.

How about the text box suggestion? I prefer keeping the thematic info in a single place, but hiding the tech part.

> For me the steps are

Step 1: check; step 2: will expand; step 3: is the majority of this part.
I disagree with the "standalone" parser: it always needs to be plugged into NOMAD or the Oasis, else it cannot read the mainfiles. Actually, I would recommend that people first connect it, since they may get confused on the origin of an error once they add their own code on top.